java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java

```diff
@@ -42,56 +37,34 @@ public class HttpKafkaProducer extends AbstractVerticle {
     private WebClient client;
     private long sendTimer;
     private int messagesSent;
-
     private CountDownLatch messagesSentLatch;
-
+    private OpenTelemetry openTelemetry;
```

Comment:
I don't think this is used anywhere, or?

Reply:
Correct. Now removed